code/drasil-docLang/lib/Drasil/DocumentLanguage/Definitions.hs

```diff
@@ -120,7 +120,6 @@ helpToRefField t si
   | Just _ <- lookupIndex t (s ^. gendefTable)          = refS $ gendefLookup      t (s ^. gendefTable)
   | Just _ <- lookupIndex t (s ^. theoryModelTable)     = refS $ theoryModelLookup t (s ^. theoryModelTable)
   | Just _ <- lookupIndex t (s ^. conceptinsTable)      = refS $ conceptinsLookup  t (s ^. conceptinsTable)
-  | Just _ <- lookupIndex t (s ^. labelledcontentTable) = refS $ labelledconLookup t (s ^. labelledcontentTable)
```

Comment:
This is the only use of `labelledconLookup`, and it was evidently unused. This function (`helpToRefField`) creates a "reference" (or, more appropriate from what I've seen use as: a _hyperlink_) for use in sentences.

Reply:
I think this kind of *functionality* is something we should use more. I'm completely fine with removing this particular 'implementation means' and replace it with something else later.